bom/pom.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.quarkiverse.googlecloudservices</groupId>
+    <artifactId>quarkus-google-cloud-services-bom</artifactId>
+    <name>Quarkus - Google Cloud Services - BOM</name>
+    <version>0.6.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <properties>
+        <google-cloud-sdk.version>16.1.0</google-cloud-sdk.version>
+        <google-cloud-graalvm.version>0.3.0</google-cloud-graalvm.version>
+        <!-- The Google Cloud BOM includes the 'android' version of Guava so we need to fix the 'jre' version.
+        Make sure to keep these two libraries compatible. -->
+        <guava.version>30.0-jre</guava.version>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <!-- Google Cloud dependencies, imported as a BOM -->
+            <!-- Warning : it will also manage the version of Guava -->
+            <dependency>
+                <groupId>com.google.cloud</groupId>
+                <artifactId>libraries-bom</artifactId>
+                <version>${google-cloud-sdk.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+            <dependency>
+                <groupId>com.google.guava</groupId>
+                <artifactId>guava</artifactId>
+                <version>${guava.version}</version>
+            </dependency>
```

Comment:
This guava constraint after the libraries-bom won't have any effect. If you want to override what the libraries-bom contains, it should be moved before its import. But given that libraries-bom imports guava-bom, it's better to import the whole guava bom. I am not sure though this is necessary at all though.

Reply:
Ah, it's the android version that the libraries-bom imports, sorry. It's ok then.